fe/fe-core/src/main/java/com/starrocks/sql/analyzer/SetStmtAnalyzer.java

```diff
@@ -211,6 +211,16 @@ private static void analyzeSystemVariable(SystemVariable var) {
             }
         }
 
+        if (variable.equalsIgnoreCase(SessionVariable.COMPUTATION_FRAGMENT_SCHEDULING_POLICY)) {
+            SessionVariableConstants.ComputationFragmentSchedulingPolicy computationFragmentSchedulingPolicy =
+                    Enums.getIfPresent(SessionVariableConstants.ComputationFragmentSchedulingPolicy.class,
+                            StringUtils.upperCase(resolvedExpression.getStringValue())).orNull();
+            if (computationFragmentSchedulingPolicy == null) {
+                String legalValues = Joiner.on(" | ").join(SessionVariableConstants.ComputationFragmentSchedulingPolicy.values());
+                throw new IllegalArgumentException("Legal values of computation_fragment_scheduling_policy are " + legalValues);
```

Comment:
maybe SematicException would be better?

Reply:
i refer to other variable like choose_execute_instances_mode , use the IllegalArgumentException . need i to update to SematicException ?

Reply:
most of the exceptions in this file are `SemanticException` for invalid input value. `choose_execute_instances_mode` is an exception. Thought it would be good to keep the same as the majority?

Reply:
i have updated it, please check and approve workflow again.